cli/js/form_data_test.ts

```diff
@@ -36,7 +36,9 @@ test(function formDataParamsGetSuccess(): void {
   formData.append("a", "true");
   formData.append("b", "false");
   formData.append("a", "null");
+  // @ts-ignore
```

Comment:
IF these are values that don't throw, then they need to be included in the interfaces, not ignored.

Reply:
I would agree but as things currently are `FormData` in `cli/js/dom_types.ts` matches the definition in TypeScript's `lib/lib.dom.d.ts`. I personally wouldn't want to see Deno's DOM types diverge from TypeScript.
Again, I could alternatively change these to `any` casts,

What do you think?

Reply:
Yeah, we don't want to deviate, so commenting again of testing contra the typings.